redwood/api/src/tasks/syncStarknetState.ts

```diff
@@ -103,21 +103,21 @@ export const importProfile = async (profileId: number) => {
     },
   })
 
-  const status = parseChallengeStatus(parseNumber(exported.status))
-  if (status !== CachedProfile.status(profileFields)) {
+  const status = parseChallengeStatus(parseNumber(exported.current_status))
+  if (status !== currentStatus(profileFields)) {
     console.warn(
       `Profile ${profileId} status mismatch. ${
-        exported.status
-      } expected, ${CachedProfile.status(profileFields)} derived`
+        exported.current_status
+      } expected, ${currentStatus(profileFields)} derived`
     )
   }
 
-  const isVerified = parseBoolean(exported.is_verified)
-  if (isVerified !== CachedProfile.isVerified(profileFields)) {
+  const isVerifiedonStarkNet = parseBoolean(exported.is_verified)
+  if (isVerifiedonStarkNet !== isVerified(profileFields)) {
```

Comment:
for this to be camel case, should the `o` be capitalized?

also we should probably standardize on e.g. `syncStarknetState` vs `syncStarkNetState`. I'd be open to either (have a very tiny leaning towards the former), I think right now we're using some of each